src/main/scala/rocks/muki/graphql/codegen/ScalaSourceGenerator.scala

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2017 Mediative
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package rocks.muki.graphql.codegen
+
+import scala.collection.immutable.Seq
+import scala.meta._
+import sangria.schema
+
+/**
+  * Generate code using Scalameta.
+  */
+case class ScalametaGenerator(moduleName: Term.Name,
```

Comment:
We are not adding comments yet to the generated class. In the future we should do that
so users have a proper scaladoc ( if they use graphql comments).

Reply:
True, let's create a ticket.

Reply:
Done in #16